packages/ui5-proxy-middleware/src/ui5/middleware.ts

```diff
@@ -37,10 +37,11 @@ function createProxyOptions(logger: ToolsLogger, config: UI5ProxyConfig): Option
  * @returns handler function
  */
 function createRequestHandler(routes: { route: string; handler: RequestHandler }[]): RequestHandler {
-    return (req, res, next) => {
+    return (req, res, next): void => {
         for (const route of routes) {
             if (req.path.startsWith(route.route)) {
-                return route.handler(req, res, next);
+                route.handler(req, res, next);
+                return;
```

Comment:
Why this change? A `RequestHander` should be returned, now it returns `undefined`.

Reply:
I does not need to return it, it just needs to call it. `createRequestHandler` returns a `RequestHandler` that checks the list of routes, and if it matches, it handles the request otherwise, it calls `next()`.
Previously the function was implicitly returning `RequestHandler | undefined` now we are explicit and made the function `void` because no one is reading the return value.